src/workerd/api/global-scope.h

```diff
@@ -382,6 +382,35 @@ struct ExportedHandler {
   jsg::Optional<jsg::Ref<ExecutionContext>> getCtx() {
     return ctx.map([&](jsg::Ref<ExecutionContext>& p) { return p.addRef(); });
   }
+
+  template <typename T>
+  static jsg::LenientOptional<T> ADDREF(jsg::LenientOptional<T>& function, jsg::Lock& js) {
```

Comment:
As a rule whenever we have a parameter `jsg::Lock& js`, we always make it the first parameter.